src/jit/importer.cpp

```diff
@@ -13351,7 +13351,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         }
                     }
 
-                    if (prefixFlags & PREFIX_UNALIGNED)
```

Comment:
Yeah, The diff does have a big discontinuity in the line numbers.  I have never seen that before.
Makes it hard to code review.